### PR TITLE
pipeline.yaml: Enable more architectures

### PR DESCRIPTION
**Pull request by nuclearcat**
Enable more architectures builds to see what is missing in kbuild.

### CODE REVIEW

File: config/pipeline.yaml [hunk 1]
```diff
@@ -180,6 +185,35 @@ jobs:
       cross_compile_compat: 'arm-linux-gnueabihf-'
       fragments: ['arm64-chromebook']
 
+  kbuild-gcc-10-arm:
+    template: kbuild.jinja2
+    kind: kbuild
```
Comment (a-wai) [inline config/pipeline.yaml:190]:
```suggestion
    <<: *kbuild-job
```

File: config/pipeline.yaml [hunk 2]
```diff
@@ -180,6 +185,35 @@ jobs:
...
+    image: kernelci/staging-gcc-10:arm64-kselftest-kernelci
+    params:
+      arch: arm
+      compiler: gcc-10
+      cross_compile: 'arm-linux-gnueabihf-'
+      defconfig: defconfig
+
+  kbuild-gcc-10-i386:
+    template: kbuild.jinja2
+    kind: kbuild
```
Comment (a-wai) [inline config/pipeline.yaml:200]:
```suggestion
    <<: *kbuild-job
```

File: config/pipeline.yaml [hunk 3]
```diff
@@ -180,6 +185,35 @@ jobs:
...
+    image: kernelci/staging-gcc-10:x86-kselftest-kernelci
+    params:
+      arch: i386
+      compiler: gcc-10
+      defconfig: i386_defconfig
+
+  kbuild-gcc-10-riscv64:
+    template: kbuild.jinja2
+    kind: kbuild
```
Comment (a-wai) [inline config/pipeline.yaml:209]:
```suggestion
    <<: *kbuild-job
```

**Review by a-wai — CHANGES_REQUESTED** [review]

**Comment by nuclearcat** [comment]
@a-wai - thanks, added anchor.
Tested on staging:
https://staging.kernelci.org:9000/viewer?node_id=65bbb61f1893bb5c371d4525
https://staging.kernelci.org:9000/viewer?node_id=65bbb6231893bb5c371d4528
https://staging.kernelci.org:9000/viewer?node_id=65bbb6211893bb5c371d4527

**Pull request merged**